home.css

```diff
@@ -1,133 +1,280 @@
-* {
+*{
     box-sizing: border-box;
     margin: 0;
     padding: 0;
+    /* font-family: Arial, Helvetica, sans-serif; */
```

Comment:
Why do you declare every font-family separated? 
And why do you declare it inside the * class?